src/test/java/edu/ucla/library/services/utility/PropsWriterTest.java

```diff
@@ -0,0 +1,126 @@
+
+package edu.ucla.library.services.utility;
+
+import static com.github.stefanbirkner.systemlambda.SystemLambda.catchSystemExit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.junit.contrib.java.lang.system.SystemErrRule;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+
+/*
+ * Unit tests for ProsWriter.java
+ */
+public class PropsWriterTest {
+
+  /**
+   * default properties file.
+  */
+    private static final String DEFAULTS =
+        "src/test/resources/properties.default";
+
+  /**
+   * output template file.
+  */
+    private static final String TEMPLATE =
+        "src/test/resources/properties.tmpl";
+
+  /**
+   * final output file.
+  */
+    private static final String OUTPUT =
+        "src/test/resources/output.properties";
+
+  /**
+   * fake file name used to cause app failures.
+  */
+    private static final String FAKE =
+        "src/test/resources/not.there";
+
+  /**
+   * rule that allows catching System.err output in tests.
+  */
+    @Rule
+    public final SystemErrRule mySystemErrRule = new SystemErrRule().enableLog();
+
+  /**
+   * Sets up a PropsWriter test.
+   *
+   * @throws IOException If there is a problem deleting the file
+  */
+    @Before
+    public void setUp() throws IOException {
+        // delete the output file if lest over from previous run
+        Files.deleteIfExists(FileSystems.getDefault().getPath(OUTPUT));
```

Comment:
This feels like it should go in an `@After` method instead, since it's a clean-up operation